app/controllers/api/v1/keywords_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class KeywordsController < ApplicationController
+      def index
+        _, keywords = pagy(KeywordsQuery.new.call)
+
+        render(json: ::V1::KeywordSerializer.new(keywords))
```

Comment:
Just a minor thing, I'm curious what error if we do not add the `::` in front of the `V1::KeywordSerializer` 

```suggestion
        render(json: V1::KeywordSerializer.new(keywords))
```

Reply:
It'll raise this error (I still don't know why)

```
NameError:
  uninitialized constant Api::V1::KeywordSerializer
```

Should I wrap the serializer under the Api namespace as well, change it from:
```ruby
module V1
  class KeywordSerializer
  end
end
```

to

```ruby
module Api
  module V1
    class KeywordSerializer
    end
  end
end
```

Reply:
Ah I see, then let keep using the `::V1::KeywordSerializer.new(keywords)` @Goose97 👍